### PR TITLE
Make the logo on the breach detail page smaller

### PR DESCRIPTION
**Pull request by Vinnl**
# References: 
Jira: MNTOR-1494
Figma: N/A

# Description

The icon is now inline wit the company title, and smaller. It's still larger than a regular favicon and so might still be a little bit blurry, but less obvious than it was.

# Screenshot (if applicable)

Blurry:

image

Sharp:

image

Fallback:

image

# How to test

Open a public breach page.

# Checklist (Definition of Done)
- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, visual change
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process. - N/A
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage. - N/A

### CODE REVIEW

**Comment by pdehaan** [comment]
Unrelated, but semi-related, but discussed previously...
I'm still seeing some very intermittent and odd issues w/ the kickstarter logo. Sometimes it's an .ico, sometimes it's a fallback "K" and sometimes it's just a broken image. I have a feeling it might be semi-different results each time if I'm hitting a different server in a kubernetes cluster on each page refresh. But not sure if that affects any fallback logic that we have where it tries to load an image that may or may not exist on a server. Or if this is all irrelevant if/when we move the breach logos onto a CDN like we did with the old assets.

**Comment by toufali** [comment]
> Unrelated, but semi-related, but discussed previously... I'm still seeing some very intermittent and odd issues w/ the kickstarter logo. Sometimes it's an .ico, sometimes it's a fallback "K" and sometimes it's just a broken image. I have a feeling it might be semi-different results each time if I'm hitting a different server in a kubernetes cluster on each page refresh. But not sure if that affects any fallback logic that we have where it tries to load an image that may or may not exist on a server. Or if this is all irrelevant if/when we move the breach logos onto a CDN like we did with the old assets.

I've been seeing similar.  From what I can tell CDN is still a ways off unless we start pushing priority of our tech-debt higher...

cc @mansaj

File: src/client/css/partials/breachDetails.css [hunk 1]
```diff
@@ -6,30 +6,32 @@
...
-  font: var(--h2-font);
+.breach-detail-meta h1 {
+  display: flex;
+  align-items: center;
+  gap: var(--padding-sm);
+}
+
+.breach-detail-meta .breach-logo {
+  height: 32px;
+  display: inline-block;
```
[the remark this answers is not in the document]
Comment (toufali) [reply src/client/css/partials/breachDetails.css:24]: Was going to say the same!  🤗

**Review by toufali — APPROVED** [review]

File: src/client/css/partials/breachDetails.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vinnl) [reply src/client/css/partials/breachDetails.css:24]: Bumped from `-sm` to `md`: c8804f7db

(But [without using margin](https://non-traditional.dev/encapsulated-css-the-key-to-composable-layouts-94f11c177cc1).)

**Comment by Vinnl** [comment]
@pdehaan could you send a screenshot of the broken image with the devtools open, if you encounter it again? And ideally, if it's the regular `` tag, what you see if you right-click to open that image in a new tab. I've seen the fallback logo for Kickstarter, but never a broken image.

**Comment by pdehaan** [comment]
> @pdehaan could you send a screenshot of the broken image with the devtools open, if you encounter it again?

I just tried in Firefox Nightly and Chrome stable and was unable to reproduce the intermittent Kickstart icon issue after 10&times; refreshes. I'll keep an eye out in the future and be sure to screenshot the Inspector/Network tabs in DevTools.

**Pull request merged**